examples/tutorials/04_cubes_and_objects/01_go_to_pose_test.py

```diff
@@ -20,21 +20,12 @@
 the given pose will assume the robot's pose as its origin.
 '''
 
-import sys
-
 import cozmo
 from cozmo.util import degrees, Pose
 
 
-def run(sdk_conn):
-    '''The run method runs once Cozmo is connected.'''
-    robot = sdk_conn.wait_for_robot()
-
+def cozmo_program(robot: cozmo.robot.Robot):
     robot.go_to_pose(Pose(100, 100, 0, angle_z=degrees(45)), relative_to_robot=True).wait_for_completed()
 
-if __name__ == '__main__':
-    cozmo.setup_basic_logging()
-    try:
-        cozmo.connect(run)
-    except cozmo.ConnectionError as e:
-        sys.exit("A connection error occurred: %s" % e)
+
+cozmo.run_program(cozmo_program)
```

Comment:
Looks like there is a weird char at the end of this line, according to the github diff? Maybe there is not a newline at the end of the file?